src/registrar/models/domain_information.py

```diff
@@ -212,36 +213,60 @@ def __str__(self):
             return ""
 
     @classmethod
-    def create_from_da(cls, domain_application, domain=None):
```

Comment:
This function is a pain everytime we need to add a field to DomainApplication, so I think we should just rip off the bandaid and refactor it to be more resilient